user_endpoints.md

```diff
@@ -1,6 +1,14 @@
 # User Endpoints
+
+## Authentication
+
 The following endpoints are only available through OAuth2 Client Credentials Flow with an application with the `provision_users` scope.
 
+The endpoints described here are considered 'application level endpoints' which is why the oauth tokens you obtain through the client credentials flow aren't tied to a user.  These tokens _cannot_ be used to access 'user level endpoints' and regular user oauth tokens cannot be used to access these application level endpoints.
```

Comment:
The endpoints described here are considered 'application level endpoints' which is why the oauth tokens you obtain through the client credentials flow aren't tied to a user.  These tokens _cannot_ be used to access 'user level endpoints' (e.g contacts, groups, etc.) and regular user OAuth tokens cannot be used to access these application level endpoints.

Reply:
We chatted about this: users can use OAuth2.